src/app/features/genes/components/gene-model-selector/gene-model-selector.component.ts

```diff
@@ -38,7 +38,11 @@ export class GeneModelSelectorComponent {
   ngOnInit() {
     this.route.queryParams.subscribe((params) => {
       const modelFromURL = params['model'];
-      const index = this._options.findIndex(o => o.value === modelFromURL);
+      let index = this._options.findIndex(o => o.value === modelFromURL);
+      if (index === -1) {
+        // default to first option if page is loaded without a model parameter
+        index = 0;
```

Comment:
although this shouldn't be the case when navigating, if the page should be opened without a parameter, we will default to the first option as per @JessterB

Reply:
Will the url be updated to include the model query param in this case?

Reply:
I decided not to since it's not an expected scenario and we will default in this case.